src/game_map.cpp

```diff
@@ -151,6 +150,7 @@ void Game_Map::Setup(int _id) {
 		events.emplace_back(location.map_id, ev);
 	}
 
+	// pan_state does not reset when you change maps.
```

Comment:
This could fix a bug in Ara Fell (have no savegame around for this) but the pan after loading a new map was completely off, which requires you to navigate blind to the next exit to fix it.